src/main/java/com/techiteasy/demo/mapping/TelevisionMapper.java

```diff
@@ -0,0 +1,75 @@
+package com.techiteasy.demo.mapping;
+
+import com.techiteasy.demo.dto.TelevisionInputDto;
+import com.techiteasy.demo.dto.TelevisionSalesDto;
+import com.techiteasy.demo.models.Television;
+import com.techiteasy.demo.dto.TelevisionDto;
+
+//Gekozen voor een aparte TelevisionMapper klasse voor een betere scheiding van verantwoordelijkheden (ipv in de service klasse)
+public class TelevisionMapper {
+
+    //Functie om Television om te zetten naar TelevisionDto
+    public static TelevisionDto toTelevisionDto(Television television) {
+        //Gebruik van var om nieuwe instantie van TelevisionDto aan te maken
+        var dto = new TelevisionDto(
+
+        // Toewijzingen naar de dto variabele
+                television.getId(),
```

Comment:
Je gebruikt hier niet de dto.

Wat je dus moet doen is dto.setId(television.getId()); etc...